drivers/overlay/overlay.go

```diff
@@ -2001,6 +2136,16 @@ func (d *Driver) ApplyDiff(id, parent string, options graphdriver.ApplyDiffOpts)
 	return directory.Size(applyDir)
 }
 
+func (d *Driver) getComposeFsPath(id string) string {
+	dir := d.dir(id)
+	return path.Join(dir, ".cfs")
```

Comment:
Is there a requirement that this directory not show up if someone runs `ls` on its parent directory?

Reply:
I don't follow.  `dir` is the parent of the layer's `diff` directory, if it has one, so `.cfs` shouldn't be visible in mounted layers?

Reply:
I've simplified the code and not create the `.cfs` subdirectory.

It is not yet added to the composefs tools, so this was just a preparatory work.  At this point it is not even clear if that is going to be used at all anyway, since overlay will support "data only" layers that are used only for metacopy operations but not to merge the content of directories.